invenio_records_marc21/services/record/utils.py

```diff
@@ -44,29 +38,51 @@ def add_file_to_record(
         file_service.commit_file(id_=marcid, file_key=filename, identity=identity)
 
 
-def create_record(service, marc21_metadata, file_path, identity):
+def create_record(service, data, file_paths, identity, do_publish=True):
     """Create the record."""
-    draft = service.create(metadata=marc21_metadata, identity=identity, files=True)
+    draft = service.create(data=data, identity=identity, files=True)
+
+    for file_path in file_paths:
+        add_file_to_record(
+            marcid=draft._record["id"],  # pylint: disable=protected-access
```

Comment:
Should  be possible to use the `marcid=draft.id` from RecordItem.[id](https://github.com/inveniosoftware/invenio-records/blob/cd622b7cf3a954912740e89f1fd145e87bedf3e9/invenio_records/api.py#L85)